core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -1409,6 +1410,32 @@ class LogManager(logDirs: Seq[File],
       None
     }
   }
+
+  def readBrokerEpochFromCleanShutdownFiles(): Long = {
+    // Verify whether all the log dirs have the same broker epoch in their clean shutdown files. If there are dir is not
+    // live, fail the broker epoch check.
+    if (liveLogDirs.size < logDirs.size) {
+      return -1L
+    }
+    var brokerEpoch = -1L
+    for (dir <- liveLogDirs) {
+        val cleanShutdownFileHandler = new CleanShutdownFileHandler(dir.getPath)
+        var textBrokerEpoch = -1L
+        try {
+          textBrokerEpoch = cleanShutdownFileHandler.read
```

Comment:
Why is this called `textBrokerEpoch`? It stores a long, not text.

Reply:
Done.